vitrina/users/migrations/0004_auto_20220909_1301.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.15 on 2022-09-09 10:01
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def fix_passwords(apps, schema_editor):
+    User = apps.get_model('vitrina_users', 'User')
+    users = User.objects.exclude(Q(password__startswith='bcrypt$') | Q(password__startswith='pbkdf2_sha256$'))
+    for user in users:
+        user.password = "bcrypt$%s" % user.password
+        user.save(update_fields=['password'])
```

Comment:
It would be nice to have a test for this.

https://github.com/wemake-services/django-test-migrations#pytest

Reply:
Tried using django-test-migrations, however since we don't have migrations for all modules, running tests gives errors unless we add --no-migrations to test command. When we add this parameter django-test-migrations tests are just skipped. I found another way to test the migration, if that is suitable.